shell/key-bindings.fish

```diff
@@ -19,14 +19,14 @@ function fzf_key_bindings
       -o -type f -print \
       -o -type d -print \
       -o -type l -print 2> /dev/null | sed 1d | cut -b3-"
-    eval $FZF_CTRL_T_COMMAND | eval (__fzfcmd) -m > $TMPDIR/fzf.result
+    eval "$FZF_CTRL_T_COMMAND |" (__fzfcmd)" -m > $TMPDIR/fzf.result"
```

Comment:
Just a minor nitpick, but I'd like the double-quotes around `(__fzfcmd)` to be symmetric.